tutorials/optical_tactile_plugin.md

```diff
@@ -53,19 +48,25 @@ The following assumptions have been made when working on this plugin:
 
 - Currently, the plugin's model should be made up of one link, one contact
 sensor and one depth camera.
-- The geometry of the sensor must be a box. Ideally, the size should be around
-20x20x5 mm, but this can be changed.
+- The geometry of the sensor must be a box, because the interpolation of the 
+contacts returned assumes 4 points of input. The visualization of contacts 
+and the `<extended_sensing>` parameter currently make this assumption as well.
+Ideally, the size should be around 20x20x5 mm, but this can be changed. That is
+the approximate size of a real tactile sensor. A bigger one will also work,
+as long as the contact surface fits inside the camera's FOV.
 - The depth camera should be displaced from the model's origin in the negative
-direction of the X axis, so it's not placed inside the membrane. This can be done
-setting the depth camera's `<pose>` accordingly. Check out the [example below](#try-it-out).
-- There should not be a `<visual>` element inside the link, so the depth camera
-isn't facing the link and can return coherent values. The 'membrane' can still
-be visualized setting the `<visualize_sensor>` parameter to true. For more information
-about the parameters, check the [parameters section](#parameters).
+direction of the X axis, so it's not placed inside the contact surface. This can
+be done by setting the depth camera's `<pose>` accordingly. Check out the [example below](#try-it-out).
+If the camera was placed in the middle of the contact surface, objects could go
+past the camera and information would be missing. 
```

Comment:
This sentence isn't clear what it's trying to say, to someone who didn't implement the plugin. Places that can be made clearer include, "in the middle" on the surface, or in the middle below the surface and inside the geometry? "Go past" in which direction - from in front of the camera to behind it, or the other way around, and when does that happen - when the sensor approaches the object?

Reply:
457691b